Issues.txt

```diff
@@ -0,0 +1,10 @@
+Issues:                                     File to start looking at?
+Does Lora Radio actually work?!?
+Fix need for serial monitor                 ??
+Get touchscreen working                     graphics/TFTDisplay.cpp
+Use power switch                            Power.cpp
+Battery level measurement                   Power.cpp
+Utilise RGB led for notifications           modules/ExternalNotificationModule.cpp AmbientLightingThread.h
+Utilise Vibration for notifications         modules/ExternalNotificationModule.cpp
+Use accelerometer/compass                   Configuration.h, main.cpp, add library
+use IR leds                                 new code!
```

Comment:
Please make sure the Todo list is not in the final commit :-)

Reply:
facepalm!